### DISCUSSION TITLE
Clarification on Team License Usage for Figma UI Kit

### DISCUSSION THREAD

**Discussion by tanssw** [primefaces/community#534] [Figma UI Kit] [answered]
I recently purchased the **Team License** for the Figma UI Kit. I'm writing to seek clarification regarding the term "Up to 5 Designers" mentioned in the license.

As I understand, the Figma UI Kit can be downloaded as a .fig file and uploaded to Figma without the need for license activation. However, I would like to ensure that I am adhering to the terms of the license and not exceeding the specified limit of 5 designers.

Could you please provide guidance on how I can track or monitor the usage to ensure compliance with the licensing terms? Additionally, if there are any specific steps or recommendations for managing the number of designers within the license limit, I would greatly appreciate your assistance.

**Comment by onursenture** [comment, marked as answer]
Hello,

Yes, as you said, we do not control license usage with a license manager. This is at your discretion. Likewise, there is no way we recommend for you to track or monitor the usage in the team license.

We trust you. Enjoy using PrimeOne! 🙏
